src/ui/handlers/run-info-ui-handler.ts

```diff
@@ -596,8 +596,13 @@ export class RunInfoUiHandler extends UiHandler {
     // Luck
     const luckValue = clamp(
       this.runInfo.party
-        .map((p) => p.toPokemon().getLuck())
-        .reduce((total: number, value: number) => (total += value), 0),
+        .map((p) => {
+          const pokemon = p.toPokemon();
+          const luck = pokemon.getLuck();
+          pokemon.destroy();
+          return luck;
+        })
+        .reduce((total: number, value: number) => total + value, 0),
```

Comment:
ah, I'm making a separate PR to fix that issue since it's completely unrelated. It should be ready later today so
you can revert the change if you want (sorry for highjacking that PR with this lol)
```suggestion
        .map((p) => p.toPokemon().getLuck())
        .reduce((total: number, value: number) => (total += value), 0),
```